proxy/envoy/discovery.go

```diff
@@ -256,12 +256,6 @@ func (ds *DiscoveryService) Register(container *restful.Container) {
 		Doc("SDS registration").
 		Param(ws.PathParameter(ServiceKey, "tuple of service name and tag name").DataType("string")))
 
-	// List all known clusters (informational, not invoked by Envoy)
-	ws.Route(ws.
-		GET("/v1/clusters").
-		To(ds.ListAllClusters).
-		Doc("Clusters in CDS"))
```

Comment:
i removed clusters temporarily. the code to list all configs needs to be cleaned up but it's useful, I get it.